libs/blocks/ost/ost.css

```diff
@@ -5,6 +5,7 @@ body.tool-ost .ost {
   left: 0;
   bottom: 0;
   right: 0;
+  margin-top: 5%;
```

Comment:
can we use a definite value here,  50px should do it.

![image](https://github.com/adobecom/milo/assets/330057/a43ce141-9360-40e9-af02-813ac7383253)

Reply:
I have updated it now.